### PR TITLE
Fix device filter form in reviewer tools (bug 1107924)

### PR DESCRIPTION
**Pull request by diox**
https://bugzilla.mozilla.org/show_bug.cgi?id=1107924

### CODE REVIEW

File: mkt/reviewers/forms.py [hunk 1]
```diff
@@ -185,6 +188,9 @@ class ApiReviewersSearchForm(ApiSearchForm):
         required=False,
         label=_lazy(u'Tarako-ready'),
         widget=CustomNullBooleanSelect)
+    dev_and_device = forms.ChoiceField(
```
Comment (diox) [inline mkt/reviewers/forms.py]: if anyone has a better name, I'm open to suggestions :)

File: mkt/reviewers/forms.py [hunk 2]
```diff
@@ -204,6 +210,14 @@ def clean_status(self):
 
         return mkt.STATUS_CHOICES_API_LOOKUP.get(status, mkt.STATUS_PENDING)
 
+    def clean(self):
+        dev_and_device = self.cleaned_data.pop('dev_and_device', '').split('+')
+        self.cleaned_data['dev'] = dev_and_device[0]
+        if len(dev_and_device) > 1:
+            self.cleaned_data['device'] = dev_and_device[1]
```
Comment (robhudson) [inline mkt/reviewers/forms.py]: It looks like this is using the slug, but the search filter is expecting the ID?

Comment (robhudson) [inline mkt/reviewers/forms.py]: Ah, nvm, I see where it subclasses `ApiSearchForm` and that's handled there.

Comment (diox) [inline mkt/reviewers/forms.py]: Yep, I'll add a short comment explaining this.

File: mkt/reviewers/forms.py (hunk 1 above)
Comment (robhudson) [inline mkt/reviewers/forms.py]: the name works for me.

**Comment by robhudson** [comment]
r+

**Pull request merged**